pandas/indexes/multi.py

```diff
@@ -864,15 +864,15 @@ def from_arrays(cls, arrays, sortorder=None, names=None):
 
         from pandas.core.categorical import _factorize_from_iterables
 
-        labels, levels = _factorize_from_iterables(arrays)
+        labels, levels = _factorize_from_iterables(arrays, dropna)
         if names is None:
             names = [getattr(arr, "name", None) for arr in arrays]
 
         return MultiIndex(levels=levels, labels=labels, sortorder=sortorder,
-                          names=names, verify_integrity=False)
+                          names=names, verify_integrity=False, dropna=dropna)
 
     @classmethod
-    def from_tuples(cls, tuples, sortorder=None, names=None):
+    def from_tuples(cls, tuples, sortorder=None, names=None, dropna=True):
         """
```

Comment:
And this is the same as [here](https://github.com/pydata/pandas/pull/14246/files#diff-1e79abbbdd150d4771b91ea60a4e1cc7L4326)